RevenueCatUI/PaywallView.swift

```diff
@@ -97,40 +95,12 @@ public struct PaywallView: View {
         )
     }
 
-    
-    /// Create a view to display the paywall with an opportunity to select a specific `Offering`.
-    ///
-    /// - Parameter offeringSelection: The closure allowing to select a specific `Offering`
-    /// containing the desired `PaywallData` to display.
-    /// - Parameter fonts: An optional `PaywallFontProvider`.
-    /// - Parameter displayCloseButton: Set this to `true` to automatically include a close button.
-    ///
-    /// - Note: Returning `nil` from the `offeringSelection` closure will fall back to displaying a default paywall.
-    /// - Note: Specifying this parameter means that it will ignore the offering configured in an active experiment.
-    /// - Warning: `Purchases` must have been configured prior to displaying it.
-    public init(
-        offeringSelection: @escaping (Offerings) -> Offering?,
```

Comment:
If I understand correctly, before we were using this to add the possibility of selecting an offer based on the user context? And now it can be removed because RC just added this, right?

Reply:
Yes exactly 👌

Reply:
Perfect, thx 👍

Reply:
not quite, they expose an API:
get offerings
create paywall with a particular offering by querying with placement-id

What Christian implemented is:
create paywall with placement-id
which internally selects offering by this id.

Reply:
They also implemented create paywall with and offeringIdentifier,
that is basically why we needed that callback